lib/esp/resources/custom_signature.rb

```diff
@@ -3,103 +3,94 @@ class CustomSignature < ESP::Resource
-    #
-    # :call-seq:
-    #  where(clauses = {})
+    #     See {API documentation}[http://api-docs.evident.io?ruby#custom-signature-attributes] for valid arguments
+    #   @return [ActiveResource::PaginatedCollection<ESP::CustomSignature>]
 
-    ##
-    # :singleton-method: find
-    # Find a CustomSignature by id
-    #
-    # ==== Parameter
-    #
-    # +id+ | Required | The ID of the custom signature to retrieve
+    # @!method self.find(id, options = {})
+    #   Find a CustomSignature by id
     #
-    # +options+ | Optional | A hash of options
+    #   *call-seq* -> +super.find(id, options = {})+
     #
-    # ===== Valid Options
+    #   @param id [Integer, Numeric, #to_i] Required ID of the custom signature to retrieve.
+    #   @param options [Hash] Optional hash of options.
+    #     ===== Valid Options
     #
-    # +include+ | The list of associated objects to return on the initial request.
+    #     +include+ | The list of associated objects to return on the initial request.
     #
-    # ===== Valid Includable Associations
+    #     ===== Valid Includable Associations
     #
-    # See {API documentation}[http://api-docs.evident.io?ruby#custom-signature-attributes] for valid arguments
-    #
-    # :call-seq:
-    #  find(id, options = {})
-
-    # :singleton-method: all
-    # Return a paginated CustomSignature list
+    #     See {API documentation}[http://api-docs.evident.io?ruby#custom-signature-attributes] for valid arguments
+    #   @return [ESP::CustomSignature]
 
-    # :singleton-method: create
-    # Create a CustomSignature
-    # :call-seq:
-    #   create(attributes={})
-    #
-    # ==== Parameter
-    #
-    # +attributes+ | Required | A hash of custom signature attributes
+    # @!method self.all
+    #   Return a paginated CustomSignature list
     #
-    # ===== Valid Attributes
-    #
-    # See {API documentation}[http://api-docs.evident.io?ruby#custom-signature-create] for valid arguments
+    #   @return [ActiveResource::PaginatedCollection<ESP::CustomSignature>]
+
+    # @!method self.create(attributes = {})
+    #   Create a CustomSignature
+    #   *call-seq* -> +super.create(attributes={})+
     #
-    # ==== Example
+    #   @param attributes [Hash] Required hash of custom signature attributes.
+    #     ===== Valid Attributes
     #
-    #  custom_signature = ESP::CustomSignature.create(description: "A test custom signature.", identifier: "AWS::IAM::001", name: "Test Signature", risk_level: "Medium")
+    #     See {API documentation}[http://api-docs.evident.io?ruby#custom-signature-create] for valid arguments
+    #   @return [ESP::CustomSignature]
+    #   @example
+    #     custom_signature = ESP::CustomSignature.create(description: "A test custom signature.", identifier: "AWS::IAM::001", name: "Test Signature", risk_level: "Medium")
 
-    # :method: save
-    # Create or update a CustomSignature
+    # @method save
```

Comment:
Looks like yard is still parsing this as expected, but `@metho` is not listed as a valid tag.  Should be `@!method`.